nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -197,7 +197,11 @@ def __init__(self, cfg: DictConfig, trainer=None):
 
         # Set the new vocabulary
         with open_dict(cfg):
-            cfg.decoder.vocabulary = ListConfig(list(vocabulary.keys()))
+            # sidestepping the potential overlapping tokens issue in aggregate tokenizers
+            if self.tokenizer_type == "agg":
+                cfg.decoder.vocabulary = ListConfig(vocabulary)
```

Comment:
Uhh - the decoder vocab is a list of dicts of paths ?

Reply:
No I believe this is the actual joint vocab, across all tokenizers. AFAIU, it's simply used to calculate the number of classes below